app/models/team_zip_export.rb

```diff
@@ -158,31 +160,39 @@ def export_assets(elements, type, directory)
       end
       file = FileUtils.touch(name).first
       File.open(file, 'wb') { |f| f.write(asset.open.read) }
+      asset_indexes[asset.id] = name
     end
+
+    asset_indexes
   end
 
   # Helper method to extract given tables to the directory
   def export_tables(elements, type, directory)
+    table_indexes = {}
+
     elements.each_with_index do |element, i|
       table = element.table
       table_name = table.name.presence || 'Table'
       table_name += i.to_s
 
       if type == :step
-        name = "#{directory}/#{handle_name(table_name)}" \
+        name = "#{directory}/#{to_filesystem_name(table_name)}" \
                "_#{i}_Step#{element.step.position + 1}.csv"
       elsif type == :result
-        name = "#{directory}/#{handle_name(table_name)}.csv"
+        name = "#{directory}/#{to_filesystem_name(table_name)}.csv"
       end
       file = FileUtils.touch(name).first
       File.open(file, 'wb') { |f| f.write(table.to_csv) }
+      table_indexes[table.id] = name
     end
+
+    table_indexes
   end
 
   # Helper method for saving inventories to CSV
   def save_inventories_to_csv(path, repo, repo_rows, id)
-    repo_name = handle_name(repo.name) + "_#{id}"
-    file = FileUtils.touch("#{path}/#{repo_name}.csv").first
+    repo_name = "#{path}/#{to_filesystem_name(repo.name)}_#{id}.csv"
```

Comment:
This is not cool. I fixed this in this [PR](https://github.com/mz3944/scinote-web/pull/1). Please accept it.